fastlane/Fastfile

```diff
@@ -515,7 +527,7 @@ platform :ios do
     FileUtils.cp(release_notes_source, File.join(metadata_directory, 'en-US', 'release_notes.txt'))
 
     git_commit(
-      path: Dir.glob('**/*.txt', base: metadata_directory),
+      path: File.join(metadata_directory, '**', '*.txt'),
```

Comment:
`Dir.glob('**/*.txt', base: metadata_directory)` didn't work. From what I understand and saw locally, there are two reasons.

First, even if we use `base:`, the paths returned by `glob` are still relative. From [the docs](https://ruby-doc.org/core-2.7.4/Dir.html#method-c-glob):

> The optional base keyword argument specifies the base directory for interpreting relative pathnames instead of the current working directory. As the results are not prefixed with the base directory name in this case, you will need to prepend the base directory name if you want real paths.

Second, `Dir.glob` returns the files that match the given pattern, but that excludes _deleted_ files. If a file is not there anymore, it can't match the given pattern. This would have made the commit incomplete.

We could argue whether our code should delete the existing translations when it doesn't find new ones. Even if we concluded that behavior is not desirable, we'd still need to account for it locally until we update it in the release toolkit. Otherwise, our local copies would remain dirty after downloading the release notes metadata if no new ones are available.

Reply:
@mokagio One thing I just thought about with using `git_commit` with `fastlane/metadata/**/*.txt` is: is there a risk that this might also commit especially the `review_information` file — which contains secret information, especially credentials that Apple reviewers should use to test the app? 🤔 

I think not, given [this file is `.gitignore`'d in the first place](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/.gitignore#L79), so I'm rather confident this is all good. But just wanted to flag this anyway as something to double-check next time you'll run `deliver` and all.
_See also p91TBi-4Le-p2#comment-4004 — which is not the same thing (the file did not get commited, but that did not prevent to being updated and changed in our local Macs during a past deliver _download_) but tangential._